src/unity.c

```diff
@@ -2329,6 +2329,17 @@ int UnityParseOptions(int argc, char** argv)
                     UnityPrint("ERROR: Unknown Option ");
                     UNITY_OUTPUT_CHAR(argv[i][1]);
                     UNITY_PRINT_EOL();
+                    /* fall-through to display help */
+                case 'h':
+                    UnityPrint("Options: "); UNITY_PRINT_EOL();
+                    UnityPrint("-l        List all tests"); UNITY_PRINT_EOL();
+                    UnityPrint("-f TEST   Only run tests with TEST in the name"); UNITY_PRINT_EOL();
+                    UnityPrint("-n TEST   Only run tests with TEST in the name"); UNITY_PRINT_EOL();
```

Comment:
I'd not include the `n` option... or if I did include it, mark it as `deprecated`?